grype/db/v6/installation/curator.go

```diff
@@ -101,12 +102,23 @@ func (c curator) Status() db.Status {
 		}
 	}
 
+	var persistentErr error
+	digest, persistentErr := db.CalculateDBDigest(c.config.DBFilePath())
+
+	if validateErr := c.validate(); validateErr != nil {
```

Comment:
How did the `digest` from line 106 get into `c.validate()`? Are we calculating the digest twice?